acceptance/localcluster/localcluster.go

```diff
@@ -127,6 +130,18 @@ func Create(numNodes int, stopper chan struct{}) *Cluster {
 		client:  newDockerClient(),
 		stopper: stopper,
 		Nodes:   make([]*Container, numNodes),
+		events:  make(chan Event, 1000),
+	}
+}
+
+func (l *Cluster) expectEvent(c *Container, msgs ...string) {
+	for index, ctr := range l.Nodes {
+		if ctr != nil && c.ID != ctr.ID {
```

Comment:
okay, but `ctr` is also never nil; that slice is populated with pointers because of ergonomics, nothing else